src/mlpack/tests/decision_tree_test.cpp

```diff
@@ -57,35 +55,37 @@ BOOST_AUTO_TEST_CASE(GiniGainEvenSplitTest)
   // Test that it's -0.5 regardless of the number of classes.
   for (size_t c = 2; c < 10; ++c)
   {
-    BOOST_REQUIRE_CLOSE(
-        GiniGain::Evaluate<false>(labels, c, weights), -0.5, 1e-5);
+    REQUIRE(GiniGain::Evaluate<false>(labels, c, weights) ==
+        Approx(-0.5).epsilon(1e-7));
+
     double weightedGain = GiniGain::Evaluate<true>(labels, c, weights);
 
     // The weighted gain should stay the same with unweight one
-    BOOST_REQUIRE_EQUAL(
-        GiniGain::Evaluate<false>(labels, c, weights), weightedGain);
+    REQUIRE(GiniGain::Evaluate<false>(labels, c, weights) == weightedGain);
   }
 }
 
 /**
  * The Gini gain of an empty vector is 0.
  */
-BOOST_AUTO_TEST_CASE(GiniGainEmptyTest)
+TEST_CASE("GiniGainEmptyTest", "[DecisionTreeTest]")
 {
   arma::rowvec weights = arma::ones<arma::rowvec>(10);
   // Test across some numbers of classes.
   arma::Row<size_t> labels;
   for (size_t c = 1; c < 10; ++c)
-    BOOST_REQUIRE_SMALL(GiniGain::Evaluate<false>(labels, c, weights), 1e-5);
+    REQUIRE(GiniGain::Evaluate<false>(labels, c, weights) ==
+        Approx(0.0).margin(1e-7));
```

Comment:
Can you use `{}` for multiple lines?

Reply:
I will make sure of this in next test I migrate